macro/run_simScript.py

```diff
@@ -150,6 +145,7 @@
 if options.DarkPhoton: HNL = False
 if not options.theMass:
   if options.DarkPhoton: options.theMass  = theDPmass
+  if options.ALPACA:     options.theMass  = theALPmass
```

Comment:
What's the point of `--mass` if it's overwritten with the default here?

Also, please avoid writing conditionals and following statements on one line. It's much more readable if you write
```python
if condition:
    expression
```

Reply:
"Choose DP production meson source(pi0,eta,omega,eta1,eta11" together with pythia8darkphoton_conf.configure, see below.

I don't think the help text explains the real meaning of the variable. It is not only selecting the mother particle but also the production mode. Particle called eta1 or eta11 is confusing, please use names from ROOT.TDatabasePDG.Instance().



    if motherMode=='pi0' and pi0mass-mass>=0.0000001:
        # use pi0 -> gamma A'
        selectedMum = 111
    elif motherMode == 'eta' and etamass-mass>=0.000001:
        # use eta -> gamma A'
        #print "eta"
        selectedMum = 221
    elif motherMode=="omega" and omegamass-mass>=0.00001:
        # use omega -> pi0 A'
        #print "omega"
        selectedMum = 223
        P8gen.SetParameters("223:oneChannel = 1 1 0 111 9900015")
        
    elif motherMode=='eta1' and eta1mass-mass>=0.00001:
        # use eta' -> gamma A'
        selectedMum = 331
        P8gen.SetParameters("331:oneChannel = 1 1 0 22 9900015")
        #should be considered also for mass < 0.188 GeV....
        #P8gen.SetParameters("331:oneChannel = 1 1 0 223 9900015")29%BR
        #P8gen.SetParameters("331:oneChannel = 1 1 0 113 9900015")2.75%BR
        
    elif motherMode=='eta11' and eta1mass-mass>=0.00001:
        # use eta' -> gamma A'
        selectedMum = 331

Reply:
Okay, I will rewrite the text and put the names from PDG database.
I need to think about the name of eta1 and eta11 since eta' has two different production modes.
I am open to suggestions :D

Reply:
Which production modes are these? that might give me some inspiration for naming.

Reply:
Dark Photon Production via mesons

Reply:
You said there are two different production modes of the eta'. What are they?

Reply:
Different thing:
Is there a reason the epsilons are not all the same?
It might be more understandable what you're doing if you define `epsilon = 0.00001` once and then test for `mass >= <meson>mass + epsilon`. Does `>` not suffice instead of the `>= small number` construction?

Reply:
having an epsilon variable would be a nice idea but they have different significant figures. So, epsilon should differ, right?

Reply:
> Different thing:
> Is there a reason the epsilons are not all the same?
> It might be more understandable what you're doing if you define `epsilon = 0.00001` once and then test for `mass >= <meson>mass + epsilon`. Does `>` not suffice instead of the `>= small number` construction?

I modified it.

Reply:
> You said there are two different production modes of the eta'. What are they?

they are;
pi0 -> photon + darkphoton
eta -> photon + darkphoton
omega -> pi0 + darkphoton
eta' -> photon + darkphoton
eta' -> rho + darkphoton

Reply:
still eta' remains as eta1 and eta11.
help message will change un run_simScript.
 I will think about them.

Reply:
Maybe they could be etaprime and etaprime_rho?